pyquotes/scrapper.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import random
+
+#website used :
+parent_link = "https://www.quoteload.com"	
+#for quote of the day :
+day_quote = "https://www.brainyquote.com/quote_of_the_day"
```

Comment:
If you are using `brainyquote` then remove I suggest you remove this because the same thing will be there in other scrapper.

Reply:
Sure. But quoteland doesn't have any section like quote of the day. So if I scrap from there, the quote won't be random. Is it ok not to have a random quote in quote of the day ?

Reply:
Remove the quote of the day section, and add random quote, we will add this feature in the API.